src/elements/callout/CalloutTable.tsx

```diff
@@ -202,41 +205,48 @@ export const CalloutTable = ({
             }
           />
         )}
-
-        {useDefaultTitle.value ? (
+        {isNonCollapsible.value && (
```

Comment:
Non-blocking: this could be refactored into a ternary operator which might make the logic easier to follow.

Also non-blocking: it would be nice to have some comments explaining the props. `isNonCollapsible` was not immediately apparent to me.